README.txt

```diff
@@ -73,11 +73,11 @@ see it miserably fail, asking for these packages, we provide the list below for
 	libxslt-devel - for valgrind
 	docbook-xsl-stylesheets - for valgrind
 	git client - for GLIBC checkout (repo migrated from CVS to git)
-	java-1_5_0-ibm-devel (for Oprofile)
+	java-1_5_0-ibm-devel - for oprofile
 	xmllint
 	lsb-release
 	bzip2-devel
-	imake or xorg-x11-utils (for OpenSSL)
+	imake or xorg-x11-utils - for openssl
 	papi-devel - for oprofile
```

Comment:
It looks like these changes are unrelated to the removal of OProfile and libpfm.  It would be better to move them to a separate commit.